src/Core/Services/MetadataProviders/SqlMetadataProvider.cs

```diff
@@ -285,6 +285,16 @@ public async Task InitializeAsync()
             _logger.LogTrace($"Done inferring Sql database schema in {timer.ElapsedMilliseconds}ms.");
         }
 
+        /// <inheritdoc />
+        public void InitializeAsync(
+            Dictionary<string, DatabaseObject> entityDatabaseObject,
```

Comment:
```suggestion
            Dictionary<string, DatabaseObject> entityDatabaseObjects,
```

Reply:
renamed to entityToDatabaseObject to match with what is the file currently